locations/spiders/redtag_fashion_ae_sa.py

```diff
@@ -0,0 +1,104 @@
+import json
+from typing import Any
+
+from scrapy import Spider
+from scrapy.http import Request, Response
+
+from locations.items import Feature
+
+
+class RedtagFashionAESASpider(Spider):
+    name = "redtag_fashion_ae_sa"
+    item_attributes = {
+        "brand": "Red Tag",
+        "brand_wikidata": "Q132891092",
+    }
+
+    def start_requests(self):
+        countries = ["Saudi Arabia", "UAE"]
+        for country in countries:
+            yield Request(
+                url=f"https://redtagfashion.com/locations/locations.php?country={country}",
+                callback=self.parse,
+                meta={"country": country},
+            )
+
+    def parse(self, response: Response, **kwargs: Any) -> Any:
+        try:
+            stores = json.loads(response.text)
+        except json.JSONDecodeError:
+            self.logger.error(f"Failed to parse JSON response: {response.text[:200]}")
+            return
+
+        for store in stores:
```

Comment:
Can be simplified to just:

```suggestion
        for store in response.json():
```

Spider will throw an error anyway if it's not able to parse JSON.